File: Example C# Sharp/Example/Test_License.cs
using System;
using System.Net;
using System.Windows.Forms;
using Newtonsoft.Json.Linq;
using static Example.CGAuth_Module;

namespace Example
{
    public partial class Test_License : Form
    {
        public Test_License()
        {
            InitializeComponent();
        }

        // Main license validation button click handler
        private void button1_Click(object sender, EventArgs e)
        {
            // Configure SSL/TLS certificate validation
            ServicePointManager.ServerCertificateValidationCallback = ValidateCert;
            // Force TLS 1.2 protocol for secure communication
            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;

            // Get the hardware ID of the current machine
            string hwid = GetHWID();
            
            // Attempt to authenticate using license key and HWID
            JObject result = AuthLicense(Text_License.Text, hwid);

            // Check if authentication was successful
            if (result["success"].ToObject<bool>())
            {
                // Extract license data from the response
                JToken data = result["data"];
                string days = data["days_remaining"].ToString();
                string hours = data["hours_remaining"].ToString();
                string app_name = data["app_name"].ToString();

                // Build success message with subscription details
                string message = $"Login successful!\n\n" +
                               $"  • App Name: {app_name}\n" +
                               $"Status: {data["status"]}\n\n" +
                               $"Time Remaining:\n" +
                               $"  • Days: {days}\n" +
                               $"  • Hours: {hours}\n";

                // Verify that the license belongs to this application
                if (app_name == YOUR_APP_NAME)
                {
                    MessageBox.Show(message, "Success", MessageBoxButtons.OK, MessageBoxIcon.Information);
                }
                else
                {
                    // License is valid but for a different application
                    MessageBox.Show("This license does not belong to this application.",
                                  "Error",
                                  MessageBoxButtons.OK,
                                  MessageBoxIcon.Error);
                }
            }
            else
            {
                // Authentication failed - show error message
                MessageBox.Show($"Login failed!\nError: {result["error"]}",
                              "Error",
                              MessageBoxButtons.OK,
                              MessageBoxIcon.Error);
            }
        }
    }
}

File: Example VB.NET/Example/Test_License.vb
Imports System.Net

Public Class Test_License
    ''' <summary>
    ''' Event handler for Submit button - processes license key authentication
    ''' </summary>
    Private Sub Button_Submit_Click(sender As Object, e As EventArgs) Handles Button_Submit.Click
        ' Configure SSL/TLS settings for secure communication
        ' Set custom certificate validation callback
        ServicePointManager.ServerCertificateValidationCallback = AddressOf ValidateCert
        ' Force TLS 1.2 protocol for secure connection
        ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12

        ' Generate unique hardware ID for this machine
        Dim hwid As String = GetHWID()
        
        ' Authenticate using license key and HWID
        Dim result = AuthLicense(Text_License.Text, hwid)
        
        ' Check if authentication was successful
        If result("success").ToObject(Of Boolean)() Then
            ' Extract authentication data from response
            Dim data = result("data")
            Dim days = data("days_remaining").ToString()
            Dim hours = data("hours_remaining").ToString()
            Dim app_name = data("app_name").ToString()

            ' Build success message with license details
            Dim message As String = "Login successful!" & vbCrLf & vbCrLf &
                               "  • App Name: " & app_name & vbCrLf &
                               "Status: " & data("status").ToString() & vbCrLf & vbCrLf &
                               "Time Remaining:" & vbCrLf &
                               "  • Days: " & days & vbCrLf &
                               "  • Hours: " & hours & vbCrLf

            ' Verify that the license belongs to this application
            If app_name = YOUR_APP_NAME Then
                ' Display success message
                MsgBox(message, MsgBoxStyle.Information, "Success")
            Else
                ' Warn user that license is for a different application
                MsgBox("This license does not belong to this application.", MsgBoxStyle.Critical, "Success")
            End If
        Else
            ' Display error message if authentication failed
            MsgBox("Login failed!" & vbCrLf &
                   "Error: " & result("error").ToString(),
                   MsgBoxStyle.Critical, "Error")
        End If
    End Sub
End Class